packages/testkit/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
```

Comment:
Can you please extend from the base ts config?

Reply:
the base ts config is in the core package not root, i noticed that other packages like style and storybook blocks don't extend from there. Do we want to create this dependency between core and testkit?

Reply:
Alright, my mistake, I though it's shared between packages. So ignore it :)